dialogueMaker/dialogueMaker.h

```diff
@@ -64,4 +69,9 @@ class Game {
     static void printAllScenes();
 
     static void runGame(std::string startSceneId);
+
+    static void addPlayer(player p);
+
+    static void printstats();
```

Comment:
should be private unless the user wants to print it.

Reply:
Stats should be displayed all the time, right?

Reply:
It doesn't use in main.cpp right?